file/file.go

```diff
@@ -9,18 +9,35 @@ import (
 )
 
 // SafeWrite writes data to a temporary file and then renames it to what's specified in path
-func SafeWrite(path string, data []byte) error {
+func SafeWrite(path string, data []byte) (err error) {
```

Comment:
Let's not modify the function declaration - we almost never use named variables in function return section.

Reply:
done